sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/query.sql

```diff
@@ -86,6 +90,81 @@ WITH combined AS (
     AS provider,
   FROM
     contextual_services.topsites_click
+  UNION ALL
+  SELECT
+    metrics.uuid.top_sites_context_id AS context_id,
+    DATE(submission_timestamp) AS submission_date,
+    'topsites' AS source,
+    IF(events[SAFE_OFFSET(0)].name = 'contile_click', 'click', 'impression') AS event_type,
+    'phone' AS form_factor,
```

Comment:
Is `phone` a standard value here? I don't know if another dataset that includes both desktop and mobile data to use as a reference... Just wondering if there's already a standard because we usually say "mobile".